openai_agent/send_email.py

```diff
@@ -0,0 +1,76 @@
+import http.client
```

Comment:
Can you move this file into captn/email/send_email.py

Reply:
Okay. I will do it.

Reply:
This is done